lib/webhooks/webhooks.js

```diff
@@ -3,7 +3,51 @@
 var crypto = require('crypto');
 var _ = require('lodash');
 var scmp = require('scmp');
-var url = require('url');
+var urllib = require('url');
+
+/**
+ * Utility function to construct the URL string, since Node.js url library won't include standard port numbers
+ *
+ * @param {URL} parsedUrl - The parsed url object that Twilio requested on your server
+ * @returns {string} - URL with standard port number included
+ */
+function buildUrlWithStandardPort(parsedUrl) {
+  let url = '';
+  const port = parsedUrl.protocol === 'https:' ? ':443' : ':80';
+
+  url += parsedUrl.protocol ? parsedUrl.protocol + '//' : '';
+  url += parsedUrl.username;
+  url += parsedUrl.password ? ':' + parsedUrl.password : '';
+  url += (parsedUrl.username || parsedUrl.password) ? '@' : '';
+  url += parsedUrl.host ? parsedUrl.host + port : '';
+  url += parsedUrl.pathname + parsedUrl.search + parsedUrl.hash;
+
+  return url;
+}
+
+/**
+ Utility function to add a port number to a URL
+
+ @param {URL} parsedUrl - The parsed url object that Twilio requested on your server
+ @returns {string} - URL with port
+ */
+function addPort(parsedUrl) {
```

Comment:
Should we include a parameter to specify a particular user defined port? Or is the function only meant to be used internally?

Reply:
Only meant to be used internally.